robot_specific_config/abb_irb2600_12_185_moveit_config/package.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>abb_irb2600_12_185_moveit_config</name>
+  <version>0.0.0</version>
+  <description>
+    <p>
+      MoveIt package for the ABB IRB 2800-12/1.85.
+    </p>
+    <p>
+      An automatically generated package with all the configuration files
+      for using the ABB IRB 2800-12/1.18 with the MoveIt Motion Planning
+      Framework.
+    </p>
+  </description>
+  <author email="andrewjshort@gmail.com">Andrew Short</author>
+  <maintainer email="andrewjshort@gmail.com">Andrew Short</maintainer>
+  <maintainer email="levi.armstrong@swri.org">Levi Armstrong (Southwest Research Institute)</maintainer>
+
+  <license>BSD</license>
+
+  <url type="website">http://wiki.ros.org/abb_irb2600_12_185_moveit_config</url>
+  <url type="bugtracker">https://github.com/ros-industrial/abb_experimental/issues</url>
+  <url type="repository">https://github.com/ros-industrial/abb_experimental</url>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <exec_depend>abb_irb2600_support</exec_depend>
+  <exec_depend>joint_state_publisher</exec_depend>
```

Comment:
The dependencies other than `abb_irb2600_support` should theoretically be moved into the `package.xml` of `abb_irb2600_support` as they are required to launch `view_robot.launch.py`.

Reply:
fixed in [0b07883](https://github.com/PickNikRobotics/abb_ros2/pull/53/commits/0b07883d7e7cbcae2bfca6f5e69eea6911340369)